index.html

```diff
@@ -233,46 +250,57 @@ <h1 class="s">Skills</h1>
       </div>
     </ul>
     <!-- Footer Form Section -->
-
-    <section class="content-4 center">
-      <h1 class="clr-blue">
-        I'm always interested in hearing about new projects, so if you'd like to
-        chat please get in touch.
-      </h1>
-    </section>
-    <section class="center">
-      <form action="https://formspree.io/f/xeqwrjjn" method="POST">
-        <input
-          type="text"
-          class="in"
-          maxlength="30"
-          name="name"
-          placeholder="&nbsp;&nbsp;&nbsp;Full name"
-          required
-        />
-        <input
-          type="email"
-          class="in"
-          name="email"
-          placeholder="&nbsp;&nbsp;&nbsp;Email address"
-          required
-        />
-        <textarea
-          name="text"
-          id=""
-          cols="30"
-          rows="10"
-          maxlength="500"
-          required
-          placeholder="
+    <footer>
+      <div class="foot-flex">
+        <section class="content-4 center">
+          <h1 class="clr-blue">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h1>
+        </section>
+        <section class="center">
+          <form action="https://formspree.io/f/xeqwrjjn" method="POST">
+            <input
+              type="text"
+              class="in"
+              maxlength="30"
+              name="name"
+              placeholder="&nbsp;&nbsp;&nbsp;Full name"
+              required
+            />
+            <br />
+            <input
+              type="email"
+              class="in"
+              name="email"
+              placeholder="&nbsp;&nbsp;&nbsp;Email address"
+              required
+            />
+            <br />
+            <textarea
+              name="text"
+              id=""
+              class="text-in"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              required
+              placeholder="
         Hey I’d like to connect to..."
-        ></textarea>
-        <img class="foot" src="imgs/footer.png" alt="illustration" />
-        <br />
-        <button class="btn btn-2 btn-3" type="submit">Get in touch</button>
-        <p class="clr-blue">blesslinjerish@gmail.com</p>
+            ></textarea>
+            <img class="foot" src="imgs/footer.png" alt="illustration" />
+            <br />
+            <button class="btn btn-2 btn-3" type="submit">Get in touch</button>
+            <p class="clr-blue hide">blesslinjerish@gmail.com</p>
+          </form>
+        </section>
+      </div>
+      <img src="/imgs/f1.png" alt="f1" class="f f1" />
+      <img src="/imgs/f2.png" alt="f2" class="f f2" />
+      <img src="/imgs/f3.png" alt="f3" class="f f3" />
+      <div id="footer-social">
         <img src="imgs/socialmedia.png" alt="social media" />
```

Comment:
- [ ] Please use separate social media icons here instead of one image. They should provide your social media links for Gitub, Linkedin, Twitter... ✔